File: baekjoon/bronze/level2/test_1225/java/Main.java
package baekjoon.bronze.level2.test_1225.java;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] arr = sc.nextLine().split(" ");

        long sum = 0;
        for (char a : arr[0].toCharArray()) {
            for (char b : arr[1].toCharArray()) {
                sum += (long) (a - '0') * (b - '0');
            }
        }
        System.out.print(sum);
    }
}

File: baekjoon/bronze/level2/test_1225/kotlin/Main.kt
package bronze.level2.test_1225.kotlin

fun main() {
    val (a, b) = readln().split(" ")

    var sum: Long = 0;
    for (i in a) {
        for (j in b) {
            sum += i.digitToInt().toLong() * j.digitToInt().toLong()
        }
    }

    print(sum)
}
